cms/envs/common.py

```diff
@@ -2814,8 +2814,9 @@
 
 BRAZE_COURSE_ENROLLMENT_CANVAS_ID = ''
 
+# pylint: disable=line-too-long
 DISCUSSIONS_INCONTEXT_FEEDBACK_URL = ''
```

Comment:
I'm disappointed that these settings aren't annotated as required by [OEP-17](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0017-bp-feature-toggles.html#). 

I know this predates this PR, but can you add some appropriate annotations?